### PR TITLE
Add additional status bars to Readme

### PR DESCRIPTION
**Pull request by hschne**
First off, great work. Most of the other Pomodoro CLIs out there are overkill. This is nice and simple, just as it should be.

I was thinking about writing a simple shell Pomodoro tracker myself, but luckily I found this :smile: 

I'm using Polybar, and I thought it might be useful for others to add a section on how to integrate `pomo.sh` with it as well.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -73,20 +73,41 @@ using notification-daemon and send-notify.  This involves pomo.sh sleeping until
...
 
+polybar
+^^^^^^
+
+Add pomo as a new polybar module to your polybar config::
+
+    [module/pomo]
+    type = custom/script
+    interval = 1
+    exec = pomo.sh clock
```
Comment (jsspencer) [inline README.rst:104]: Can also add `click-left = pomo.sh pause` if you like.

**Comment by jsspencer** [comment]
Thanks for adding this -- I am also using polybar now and have been meaning to add docs on this (I use `pomo.sh status` with `tail = true` but it's essentially the same as your version).

Thanks for your kind words. I'm glad you are finding it useful. This reminds me that I some minor local changes that I need to push...

File: README.rst (hunk 1 above)
Comment (hschne) [reply README.rst:104]: That's a great idea! :+1:

Comment (jsspencer) [reply README.rst:104]: Do you want to add it? If so, I'll hold off merging until you do. Otherwise I can add it later.

Comment (hschne) [reply README.rst:104]: I just added it :)

**Pull request merged**